addon/components/high-charts.js

```diff
@@ -58,15 +59,16 @@ export default Ember.Component.extend({
   },
 
   draw() {
-    let options = [ get(this, 'buildOptions') ];
-    let mode  = get(this, 'mode');
+    let completeChartOptions = [ get(this, 'buildOptions'), get(this, 'callback') ],
```

Comment:
Since this is ember addon, I prefer using  [Ember.js JavaScript Style Guide](https://github.com/emberjs/ember.js/blob/master/STYLEGUIDE.md).

@huiyuan so, please update your code to follow those rules before I can merge, especially about [variables](https://github.com/emberjs/ember.js/blob/master/STYLEGUIDE.md#variables).